pkg/eventing/recorder.go

```diff
@@ -0,0 +1,109 @@
+package eventing
```

Comment:
We already have recorder library in `csi-baremetal` repo. Can it be used instead of creating new one?

Reply:
Replaced by lib in csi-baremetal repo